contrib/brl/bbas/bhdfs/pro/processes/bhdfs_fs_manager_processes.cxx

```diff
@@ -12,8 +12,8 @@
 
 namespace bhdfs_create_fs_manager_process_globals
 {
-  const unsigned n_inputs_ = 2;
-  const unsigned n_outputs_ = 1;
+  constexpr unsigned n_inputs_  = 2;
```

Comment:
The added space thing appears many times.

Reply:
Fixed.